test/e2e/assignments.txt

```diff
@@ -0,0 +1,116 @@
+**************** Sig assignments for tests ********************
+monitoring.go ('sig-ui', 2)
+deployment.go ('sig-scheduling', 4)
```

Comment:
There are a bunch of misfires here.  I think we need to let the sigs create the directory structure of have a e2e champion own the routing.